components/02-elements/tab/_tab.scss

```diff
@@ -1,99 +1,130 @@
             order: -1;
-            margin-right: $tab__margin;
-            background-color: $tab__title-background;
-            border-bottom: 0 none;
-            bottom: -1px;
+            border-top: none;
+            border-bottom : $tab__title-border;
+            font-weight: $tab__title-font-weight;
         }
 
         &--active {
+            border-top: none;
+            border-bottom: none;
+            background: $tab__title-background--mobile-active;
+            color: $tab__title-color-mobile-active;
+
+            &:nth-last-child(2) {
+                border: none;
+
+                @include mq($screen-l) {
+                    border-bottom: $tab__title-border--active;
+                }
+            }
             @include mq($screen-l) {
-                border-color: $tab__title-border-color--active;
-                border-width: $tab__title-border-width--active;
-                border-style: $tab__title-border-style--active;
+                border-bottom: $tab__title-border--active;
                 background-color: $tab__title-background--active;
+                font-weight: $tab__title-font-weight--active;
+                color: $tab__title-color;
                 z-index: 2;
             }
+            .tab__icon {
+                transform: translateY(-50%)rotateX(180deg);
 
-            .tabs__icon {
-                transform: rotate(45deg);
-
-                @include mq($screen-l) {
-                    display: none;
+                svg {
+                    fill: $color-primary;
                 }
             }
         }
     }
 
-    &__icon {
-        position: absolute;
-        top: 17px;
-        right: 10px;
-        width: $tab__icon-width;
-        height: $tab__icon-height;
-        fill: $tab__icon-fill;
-        transition: $tab__icon-transition;
-
-        @include mq($screen-l) {
-            display: none;
-        }
-    }
-
     &__content {
         position: relative;
         display: none;
         width: 100%;
-        padding: 10px;
+        padding: $tab__content-padding--mobile;
         z-index: -1;
         opacity: 0;
-        overflow-y: hidden;
-        transition: $tab__content-transition;
         overflow: hidden;
 
         @include mq($screen-l) {
+            padding: $tab__content-padding;
             flex-basis: 100%;
             z-index: 1;
         }
 
         &--active {
             display: block;
             opacity: 1;
-            border-bottom: $tab__title-border;
-            @include mq($screen-l) {
-                border-bottom: 0 none;
-                border-top: $tab__title-border;
+
+            &:nth-last-child(1) {
+                border-bottom: $tab__title-border--mobile;
+
+                @include mq($screen-l) {
+                    border-bottom: none;
+                }
             }
         }
     }
+
+    &__icon {
+        position: absolute;
+        right: 15px;
+        top: 50%;
+        transform: translateY(-50%);
```

Comment:
Let's use:
```
top: 0;
bottom: 0;
margin: auto;
```
it's a better way of vertical aligning of elements with known size.

Reply:
I would like to but...there is a slight problem :(
![image](https://user-images.githubusercontent.com/12099511/32438109-abc4cf98-c2e9-11e7-8f82-df700733268c.png)
and if I disable `position: absolute` :
![image](https://user-images.githubusercontent.com/12099511/32438126-bb7ed49c-c2e9-11e7-98c0-70a5645f9d32.png)
What do you think, Bartek ?

Reply:
It works, you just have to remove the `translateY` from modificator and keep the `right` property.
![](https://i.imgur.com/IipFiJz.png)

Reply:
you're right. Done